mobile/src/core/utils/balance/typings.ts

```diff
@@ -0,0 +1,8 @@
+import {Transaction} from '@burstjs/core';
+
+export interface BalanceHistoryItem {
```

Comment:
or even that util file, why not

Reply:
This whole file was copy/pasted from `web/angular-wallet`, so I may leave it since the suggestions are minor in exchange for keeping the files identical until we decide what to do with them. I don't think they belong in burstjs, so I figured cloning them is ok for now.